api.yml

```
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials/issueCredential:
```

-1 to verbs in URLs that have direct analogues to HTTP verbs... this has been said multiple times in this PR and others.

https://restfulapi.net/resource-naming/

There is an entire dissertation that's written about why this is a bad idea and breaks many of the design principles behind HTTP:

https://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm

---

I agree, lets discuss here: https://github.com/w3c-ccg/vc-issuer-http-api/issues/41

---

Can I resolve this?

---

So -- no discussion happened. Can we make this:

```suggestion
  /credentials:
```